roles/rsyslog/roles/input_roles/basics/defaults/main.yaml

```diff
@@ -14,6 +14,8 @@ rsyslog_unprivileged: False
 rsyslog_capabilities: []
 
 rsyslog_use_imuxsock: False
+rsyslog_input_log_path: "/var/log/containers/*.log"
+rsyslog_input_log_tag: "container"
```

Comment:
Are those internal variables, or variables expected to be configured by users?

Reply:
They need to be configured by users.

Reply:
Shouldn't it be added to the documentation then?

Reply:
> Shouldn't it be added to the documentation then?

Done.
https://github.com/linux-system-roles/logging/pull/84/files#diff-1c7bee374b17a63912b35d50adb51f95R298-R301